pennylane_lightning/lightning_qubit/lightning_qubit2.py

```diff
@@ -30,7 +25,7 @@
 
 
 try:
-    # pylint: disable=import-error, no-name-in-module
+    # pylint: disable=import-error, no-name-in-module, unused-import
     from pennylane_lightning.lightning_qubit_ops import (
```

Comment:
Could we save a few lines here?
```suggestion
    import pennylane_lightning.lightning_qubit_ops
```

Reply:
Do you mean importing only the full binaries for checking, and not the Classes?